apps/sms/style/sms.css

```diff
@@ -232,6 +231,15 @@ form[role="dialog"][data-type="edit"] .bb-tablist > li {
   pointer-events: none;
 }
 
+[data-type="list"] .summary {
```

Comment:
Yeah, I'm okay with single selector as well, I just meant that don't really like '[data-type="list"] part, it looks too generic while it's for thread list only. Maybe it's better to rename to smth like ".threadlist-item-summary" as oocss evangelizes. But up to you :)